<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1036</width>
                <height>804</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
                <spacer name="verticalSpacer_5">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>25</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QLabel" name="Header_Label">
                    <property name="font">
                        <font>
                            <family>Rockwell</family>
                            <pointsize>26</pointsize>
                            <bold>true</bold>
                        </font>
                    </property>
                    <property name="text">
                        <string>Encrpyption System</string>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="verticalSpacer_4">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>20</width>
                            <height>25</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QFrame" name="frame">
                    <property name="minimumSize">
                        <size>
                            <width>150</width>
                            <height>0</height>
                        </size>
                    </property>
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                        <item>
                            <spacer name="verticalSpacer_3">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="verticalLayout">
                                <item>
                                    <widget class="QLabel" name="Encrypt_Label">
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>18</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>Enter the text :</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QTextEdit" name="Encrypt_Text"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QVBoxLayout" name="verticalLayout_2">
                                <item>
                                    <widget class="QLabel" name="label_2">
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>18</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>Hashed Text :</string>
                                        </property>
                                        <property name="alignment">
                                            <set>Qt::AlignCenter</set>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QTextEdit" name="Hash_Text"/>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QPushButton" name="MD5_Encrypt">
                                        <property name="minimumSize">
                                            <size>
                                                <width>120</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>10</pointsize>
                                            </font>
                                        </property>
                                        <property name="cursor">
                                            <cursorShape>PointingHandCursor</cursorShape>
                                        </property>
                                        <property name="text">
                                            <string>MD5</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="SHA_3">
                                        <property name="minimumSize">
                                            <size>
                                                <width>120</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>10</pointsize>
                                            </font>
                                        </property>
                                        <property name="cursor">
                                            <cursorShape>PointingHandCursor</cursorShape>
                                        </property>
                                        <property name="text">
                                            <string>SHA -3</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="Blake_2">
                                        <property name="minimumSize">
                                            <size>
                                                <width>120</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>10</pointsize>
                                            </font>
                                        </property>
                                        <property name="cursor">
                                            <cursorShape>PointingHandCursor</cursorShape>
                                        </property>
                                        <property name="text">
                                            <string>Blake-2</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="SHA_256">
                                        <property name="minimumSize">
                                            <size>
                                                <width>120</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>10</pointsize>
                                            </font>
                                        </property>
                                        <property name="cursor">
                                            <cursorShape>PointingHandCursor</cursorShape>
                                        </property>
                                        <property name="text">
                                            <string>SHA 256</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="Btn_Clear">
                                        <property name="minimumSize">
                                            <size>
                                                <width>120</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>10</pointsize>
                                            </font>
                                        </property>
                                        <property name="cursor">
                                            <cursorShape>PointingHandCursor</cursorShape>
                                        </property>
                                        <property name="text">
                                            <string>Clear</string>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QPushButton" name="Btn_Close">
                                        <property name="minimumSize">
                                            <size>
                                                <width>120</width>
                                                <height>30</height>
                                            </size>
                                        </property>
                                        <property name="font">
                                            <font>
                                                <family>Rockwell</family>
                                                <pointsize>10</pointsize>
                                            </font>
                                        </property>
                                        <property name="cursor">
                                            <cursorShape>PointingHandCursor</cursorShape>
                                        </property>
                                        <property name="text">
                                            <string>Close</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <spacer name="verticalSpacer">
                                <property name="orientation">
                                    <enum>Qt::Vertical</enum>
                                </property>
                                <property name="sizeHint" stdset="0">
                                    <size>
                                        <width>20</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </spacer>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QFrame" name="frame_2">
                    <property name="frameShape">
                        <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                        <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                        <property name="labelAlignment">
                            <set>Qt::AlignCenter</set>
                        </property>
                        <property name="formAlignment">
                            <set>Qt::AlignCenter</set>
                        </property>
                        <property name="leftMargin">
                            <number>200</number>
                        </property>
                        <item row="1" column="0">
                            <widget class="QSplitter" name="splitter">
                                <property name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLabel" name="Hash_Identify">
                                <property name="text">
                                    <string/>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
